jobs/env/ci-kubernetes-e2e-gce-scale-performance.env

```diff
@@ -9,6 +9,9 @@ KUBE_GCI_VERSION=cos-stable-63-10032-71-0
 # Turn off advanced audit logging to simulate production
 ENABLE_APISERVER_ADVANCED_AUDIT=false
 
+# Increase master disk size to see if it helps with #62064.
+MASTER_DISK_SIZE=500GB
```

Comment:
Shouldn't we wait for one run after bumping etcd?

Reply:
That's what I thought so initially. But I think this will help us save some time (also potentially give a green run finally).
Besides, @krzysied was already running density test against 3.2.sth yesterday and that seemed to still fail with high latency.